### PR TITLE
RavenDB-13272 Refactor Export to CSV Buttons in Collection & Query views

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/query/query.ts [hunk 1]
```diff
@@ -1132,20 +1132,27 @@ class query extends viewModelBase {
...
 
-        let args: { format: string, debug?: string };
+        let args: { format: string, debug?: string, field?: string[] };
+
         if (this.criteria().indexEntries()) {
-            args = {
-                format: "csv",
-                debug: "entries"
-            };
+            args = columns ? { format: "csv", debug: "entries",field: columns } : { format: "csv", debug: "entries"};
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: args = { format: "csv", debug: "entries",field: columns }

if columsn will be undefined it won't be send to server anyway

File: src/Raven.Studio/typescript/viewmodels/database/query/query.ts [hunk 2]
```diff
@@ -1132,20 +1132,27 @@ class query extends viewModelBase {
...
         } else {
-            args = {
-                format: "csv"
-            };
+            args = columns ? {format: "csv", field: columns } : { format: "csv" };
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: same here

**Review by ml054 — APPROVED** [review]

File: src/Raven.Studio/typescript/viewmodels/database/query/query.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: done

File: src/Raven.Studio/typescript/viewmodels/database/query/query.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/query/query.ts]: done

**Review by ml054 — APPROVED** [review]

**Pull request merged**